tests/agent/test_agent.py

```diff
@@ -152,6 +170,12 @@ def test_gx_agent_gets_env_vars_on_init(get_context, gx_agent_config):
     assert agent._config == gx_agent_config
 
 
+def test_gx_agent_incorrect_token(monkeypatch):
+    monkeypatch.setenv("GX_CLOUD_ACCESS_TOKEN", "incorrect_token")
```

Comment:
```suggestion
    monkeypatch.setenv("GX_CLOUD_ACCESS_TOKEN", "invalid_token")
```
If error'ing bc expecting uuid, I'd make this more clear